packages/manager/apps/nasha-react/src/pages/details/[serviceName]/suscriptions.tsx

```diff
@@ -1,34 +1,192 @@
-import { serviceInfos } from '../../../api/nasha-react';
+import {
+  OsdsText,
+  OsdsDivider,
+  OsdsLink,
+  OsdsIcon,
+  OsdsChip,
+} from '@ovhcloud/ods-stencil/components/react/';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+} from '@ovhcloud/ods-theming';
+import { OdsIconName, OdsIconSize } from '@ovhcloud/ods-core';
+import { serviceInfos } from '@/api/nasha-react';
 
 function Subscriptions(props: { serviceName: string }) {
   const { serviceName } = props;
   const { isLoading, isError, data } = useQuery(
     ['serviceInfos', { serviceName }],
     serviceInfos,
   );
-
-  if (isLoading) {
-    return <span>Loading...</span>;
-  }
-
-  if (isError) {
-    return <span>Error...</span>;
-  }
-
   const count = data?.length;
   if (count === 0) {
     return <></>;
   }
+
+  if (isLoading) return <span>Loading...</span>;
+  if (isError)
+    return (
+      <span>Une erreur est survenue lors du chargement des données...</span>
+    );
+
+  const {
+    creation,
+    expiration,
+    engagedUpTo,
+    renew,
+    contactAdmin,
+    contactTech,
+    contactBilling,
+  } = data || {};
+
+  const dateOptions: Intl.DateTimeFormatOptions = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  };
+  const SuscriptionDateOptions = { dateOptions };
+
+  const dateCreation = creation
+    ? new Date(creation).toLocaleDateString(
+        undefined,
+        SuscriptionDateOptions.dateOptions,
+      )
+    : '';
+  const dateExpiration = expiration
+    ? new Date(expiration).toLocaleDateString(
+        undefined,
+        SuscriptionDateOptions.dateOptions,
+      )
+    : '';
+  const dateEngagementUpTo = engagedUpTo
+    ? new Date(engagedUpTo).toLocaleDateString(
+        undefined,
+        SuscriptionDateOptions.dateOptions,
+      )
+    : '';
+
+  let engagedUpToContent;
+  if (!dateEngagementUpTo || dateEngagementUpTo === 'Invalid Date') {
+    engagedUpToContent = (
+      <div>
+        <OsdsChip>Aucun engagement</OsdsChip>
+        <OsdsDivider separator={true} color={OdsThemeColorIntent.default} />
+      </div>
+    );
+  } else if (dateEngagementUpTo) {
+    engagedUpToContent = (
+      <div>
+        <OsdsText>Terminé le {dateEngagementUpTo}</OsdsText>
+        <OsdsDivider separator={true} color={OdsThemeColorIntent.default} />
+      </div>
+    );
+  }
+
   return (
     <>
-      <ul>{JSON.stringify(data)}</ul>
-      <ul>
-        <button>Configurer le renouvellement</button>
-        <button>Anticiper le paiement</button>
-        <button>Configurer le renouvellement</button>
-      </ul>
+      <div className="element">
+        <div>
+          <div>
+            <OsdsText
+              level={OdsThemeTypographyLevel.subheading}
+              color={OdsThemeColorIntent.text}
+            >
+              Date de création:
+            </OsdsText>
+          </div>
+          <OsdsText>Le {dateCreation}</OsdsText>
+        </div>
+        <OsdsDivider
```

Comment:
Cleanup props on OsdsDivider (separator={true} => separator and not need to put the color if it's the default)